index.js

```diff
@@ -1,3 +1,10 @@
 'use strict'; // eslint-disable-line strict
 
+/**
+ * Catch uncaught exceptions and add timestamp to aid debugging
+ */
+process.on('uncaughtException', err => {
+    process.stderr.write(`${new Date().toISOString()} ${err.stack}`);
```

Comment:
Since the exception backtrace is multi-line, I'd recommend for readability to add short context info around the timestamp, followed by a newline, then the exception backtrace. This way the first backtrace line is aligned with the other lines.

Here's what I'm thinking about:
```
process.stderr.write(`${new Date().toISOString()}: Uncaught exception: \n${err.stack}`); 
```